backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/nodes/AbstractNodeDescriptionProvider.java

```diff
@@ -56,4 +58,17 @@ protected void orderToolSectionsTools(List<NodeToolSection> toolSections) {
             toolSection.getNodeTools().addAll(sortedNodeTools);
         });
     }
+
+    protected ImageNodeStyleDescription createImageNodeStyleDescription(String shapeId) {
+        return this.createImageNodeStyleDescription(shapeId, this.colorProvider.getColor("transparent"));
+    }
+
+    protected ImageNodeStyleDescription createImageNodeStyleDescription(String imagePath, UserColor borderColor) {
+        return this.diagramBuilderHelper.newImageNodeStyleDescription()
+                .borderColor(borderColor)
+                .borderRadius(0)
+                .positionDependentRotation(true)
```

Comment:
This parameter is only for _rotatable_ nodes, mostly used for border nodes. Do you really need to set `true` as default and non-changeable value for all image nodes that will reuse this method?

Reply:
Changed. By default, this parameter is `false` and only set to `true` for Border nodes.